src/features/expense/ui/DayExpensesList/DayExpensesList.tsx

```
@@ -0,0 +1,23 @@
+const DayExpensesList = () => {
+  // const month = useExpenseStore((state) => state.month);
+  const dayExpenses: DayExpenses[] = useExpenseStore(
+    (state) => state.dayExpenses
+  );
+
+  // const expenseList = ;
+
+  return (
+    <div className='bg-gray-100'>
+      {dayExpenses.map(({ id, day, expenses }: DayExpenses) => (
+        <DayExpensesItem id={id} day={day} expenses={expenses} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)